doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,423 @@
+(solving-guide-inequalities)=
+# Solve an Ordinary Differential Equation (ODE) Algebraically
+
+Use SymPy to solve an ordinary differential equation (ODE) algebraically. For
+example, solving $y''(x) + 9y(x)=0 $ yields $ y(x)=C_{1} \sin(3x)+ C_{2}
+\cos(3x)$.
+
+## Alternatives to Consider
+- To numerically solve a system of ODEs, use a [SciPy ODE
+  solver](https://docs.scipy.org/doc/scipy/reference/integrate.html#solving-initial-value-problems-for-ode-systems)
+  such as `solve_ivp`. You can also use SymPy to create and then
+  {func}`~.lambdify` an ODE to be solved numerically using SciPy's as
+  `solve_ivp` as described below in [](#numerically-solve-an-ode-in-scipy).
+
+## Solve an Ordinary Differential Equation (ODE)
+
+Here is an example of solving an ordinary differential equation algebraically
+using {func}`~.dsolve`:
+
+```py
+>>> from sympy import Function, dsolve, Derivative
+>>> from sympy.abc import x
+>>> y = Function('y')
+>>> result = dsolve(Derivative(y(x), x, x) + 9*y(x), y(x))
+>>> result
+Eq(y(x), C1*sin(3*x) + C2*cos(3*x))
+```
+
+You can then use SymPy to verify that the solution is correct:
+
+```py
+>>> from sympy import checkodesol
+>>> solution = result.rhs
+>>> solution
+C1*sin(3*x) + C2*cos(3*x)
+>>> checkodesol(Derivative(y(x), x, x) + 9*y(x), solution)
+(True, 0)
+```
+
+The output of {func}`~.checkodesol` is a tuple where the first item, a boolean,
+tells whether substituting the solution into the ODE results in `0`, indicating
+the solution is correct.
+
+## Guidance
```

Comment:
I'm a little unclear how the "Guidance" section is supposed to work on these pages. How come on this page the subsequent sections are on the same level as "Guidance", but on the "solving systems of equations numerically" page several sections are under "Guidance"?

Reply:
The intent of the "Guidance" section is to have zero or more subsections that provide tips on how to set up using the function in question. 

- On this ODE page, I identified only one tip, so there's just one subsection, "Input Format"; so "Guidance" actually has one child section. The subsequent sections are not (in my way of thinking) that kind of guidance, so they're at the same level (`h2`) as "Guidance".
- On the ["solving systems of equations numerically" page](https://docs.sympy.org/dev/guides/solving/solve-numerically.html), I identified several such tips, so they're all children sections of "Guidance".

The (`h2`) sections after "Guidance" are typically "Use the Solution Result", "Tradeoffs", etc. I admit it's a somewhat subjective decision what counts as Guidance. I used a template with sections and slotted content into what I thought was the most logical section. We can revisit if needed.